app/controllers/posts_controller.rb

```diff
@@ -12,4 +12,27 @@ def show
     @user_name = User.find(@user_id).name
     puts @post.comments.to_json
   end
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @post = Post.new(post_params)
+    @post.author_id = current_user.id
+    @post.likes_counter = 0
+    @post.comment_counter = 0
+
+    if @post.save
+      redirect_to users_path
+    else
+      render :new
+    end
+  end
```

Comment:
- [ ] I can not create a post, first the URL is wrong, also the post somehow is not creating even when I change the URL to take a user id of 1, kindly watch this [video](https://www.loom.com/share/0dbc0192067c443cb4aca1e23d032373?sid=3b89c7b4-2313-40d7-a025-b29b5a69ca0e) to understand what I am saying

Reply:
I Fixed the redirect issue thanks @VICTOR-CHUKWU

Reply:
- [ ] Good job implementing the requested change by the previous reviewer, however creating a post is still not possible as there is a typo `undefined method` `comment_counter=' causing an error, kindly fix it.
![image](https://github.com/UbdaNam/Blogiverse/assets/81932478/3705d7ef-c091-4c4c-aa5e-f266f5a2fd0d)